Lib/asyncio/staggered.py

```diff
@@ -42,8 +43,6 @@ async def staggered_race(coro_fns, delay, *, loop=None):
         delay: amount of time, in seconds, between starting coroutines. If
             ``None``, the coroutines will run sequentially.
 
-        loop: the event loop to use.
-
```

Comment:
Put this back (with a deprecation note perhaps)?

Reply:
Added it back but I don't think we can add deprecation at this point for 3.13?